### PR TITLE
Release Firestore emulator 1.18.1

### PR DESCRIPTION
**Pull request by clairekeer1997**
### Description

This PR updates Firestore Emulator to v1.18.1.

### Scenarios Tested
Started emulators and it downloads and verifies the checksum correctly.
```
node firebase.js emulators:start
i  emulators: Starting emulators: firestore
i  firestore: downloading cloud-firestore-emulator-v1.18.1.jar...
Progress: ===========================================================================================================> (100% of 65MB
i  firestore: Firestore Emulator logging to firestore-debug.log
✔  firestore: Firestore Emulator UI websocket is running on 9150.
i  ui: downloading ui-v1.11.6.zip...
Progress: ============================================================================================================> (100% of 4MB
i  ui: Emulator UI logging to ui-debug.log

```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Release Firestore emulator 1.18.1 which addes a emulator configuration to start with experimental mode.
```
Comment (joehan) [inline CHANGELOG.md:1]:
```suggestion
- Release Firestore emulator 1.18.1 which addes a emulator configuration to start with experimental mode.
```

**Review by joehan — APPROVED** [review]
LGTM with one suggestion

**Review by christhompsongoogle — APPROVED** [review]

**Comment by joehan** [comment]
@christhompsongoogle ran the integration tests locally and confirmed that they are passing.

**Comment by christhompsongoogle** [comment]
Thanks Joe, the PR with the passing tests is here:

https://github.com/firebase/firebase-tools/pull/6027

**Pull request merged**